couchbase/README.md.jinja

```diff
@@ -77,8 +73,13 @@ db.createUser( {
 =======
 Couchbase provides Role-Based Access Control (RBAC) to restrict system access. A Couchbase role permits one or more resources to be accessed according to defined privileges.
 
-The [`data_monitoring` role](https://docs.couchbase.com/server/6.6/learn/security/roles.html#data-monitor) has the minimum permissions required to use the integration. For
-more information on the associations of privileges to resources and roles, see [Roles.](https://docs.couchbase.com/server/6.6/learn/security/roles.html)
+```
+db.createUser( {
+  user: "collectd",
+  pwd: "collectd",
+  roles: [ { role: "readAnyDatabase", db: "admin" }, { role: "clusterMonitor", db: "admin" } ]
+});
+```
```

Comment:
what's the context for this? the preceding line doesn't indicate exactly what this is.